packages/expo-camera/android/src/main/java/expo/modules/camera/CameraViewModule.kt

```diff
@@ -139,11 +144,51 @@ class CameraViewModule : Module() {
     OnDestroy {
       try {
         moduleScope.cancel(ModuleDestroyedException())
-      } catch (e: IllegalStateException) {
+      } catch (_: IllegalStateException) {
         Log.e(TAG, "The scope does not have a job in it")
       }
     }
 
+    Class("Picture", PictureRef::class) {
+      Property("width") { picture: PictureRef ->
+        picture.ref.width
+      }
+
+      Property("height") { picture: PictureRef ->
+        picture.ref.height
+      }
+
+      AsyncFunction("savePictureAsync") { picture: PictureRef, options: SavePictureOptions?, promise: Promise ->
+        val image = picture.ref
+        val response = Bundle()
+
+        cacheDirectory.let {
+          response.apply {
+            putInt("width", image.width)
+            putInt("height", image.height)
+          }
+
+          val quality = options?.quality ?: 0
```

Comment:
```suggestion
          val quality = options?.quality ?: 1
```
I think it would make more sense to set the default quality to 1 🤔

Reply:
I thought that too and realised it's been 0 for a long time. I'll change it on both. It's not breaking